### ISSUE TITLE
increase default checkpoint intervall for 1.0.x?

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#2841]
from irc:
```
22:42  izibi$ I hope you can imagine my surprise when I figured out borg wrote ~700GB
              to the local disk when looking why the traffic on the repo host would pause for 
              2.5 minutes every 5 minutes
22:43  ThomasWaldmann$ heh, yeah
22:43  ThomasWaldmann$ maybe we should increase the --checkpoint-interval also for 1.0?
22:44  izibi$ probably a good idea
```

**Comment by ThomasWaldmann** [comment]
pro: maybe more sane default behaviour

contra: changing defaults in a patch release

**Comment by ThomasWaldmann** [comment]
default in 1.0 is 300s, default in 1.1 will be 1800s.

**Comment by enkore** [comment]
I'd say it's reasonable to bump 1.1 to 3600 s (1 hour).

**Comment by jdchristensen** [comment]
Can the checkpointing be made efficient enough that we can keep the checkpoint interval under, say, 20 minutes without causing too much overhead?  It's not unusual for network connections to hang, and checkpointing is great for saving the part that has been done.

**Comment by ThomasWaldmann** [comment]
Somehow the idea of 30mins was "not too long, so people don't lose too much, but also not too short, keep overhead low".

Of course the sweet spot here depends a lot on the circumstances. That's why we have a commandline option. The default should address something that usually works for most people.

Also, this ticket was about changing the 1.0.x default, not about master. :D

**Comment by ThomasWaldmann** [comment]
fixed by #2843.

**Comment by enkore** [comment]
> Can the checkpointing be made efficient enough that we can keep the checkpoint interval under, say, 20 minutes without causing too much overhead?

No, the way transactions are implemented do not permit efficient implementation of frequent commits on principle.
